oidc-provider/simple_op/src/provider/authn/__init__.py

```diff
@@ -0,0 +1,30 @@
+import importlib
```

Comment:
I agree with @jbu --- since this is a direct copy of pyoidc I don't think we should edit it in any way. I'm not hugely keen on downloading a new copy each time, as this will inevitably break some time in the future. At least if we have the code checked in here, we're all working from the same auth provider code for development. 

I think we should squash the commits on this and merge ASAP.